src/ResourceManager/EventGrid/Commands.EventGrid/EventSubscription/NewAzureEventGridSubscription.cs

```diff
@@ -280,12 +280,122 @@ public class NewAzureEventGridSubscription : AzureEventGridCmdletBase
         [ValidateNotNullOrEmpty]
         public string[] Label { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            Position = 9,
```

Comment:
Remove position from the new parameters - no new non-mandatory parameters should have a position.

Reply:
done